### ISSUE TITLE
UK Adjectival grading system not handled correctly.

### ISSUE THREAD

**Issue by scd** [theCrag/website#2535]
Internally the UK Adjectival system is labelled as UK system. This is not a great label as the technical grade is the main system, however changing this is very problematic.

Here is a list of things which are wrong with the way we handle UK (Adjectival) system
 - [x] should be removed from grade conversions table
 - [ ] consider creating a separate type (rather than 'Free') for UK system. There are a number of other systems that could be converted to this new type, such as Brazil General grade. This would need considerable investigation to make sure there are no regression issues.
 - [x] should not be included in CPR calculations
 - [x] should not be included in route difficulty assessment as this is technical focused.
 - [x] there is a further bug in the route difficulty assessment where the UK Technical grade is assigned to the secondary difficulty value. This is just wrong. When fixing this we should use the new CPRPriorityFlag to assign the primary assessment value.

**Comment by scd** [comment]
I have done this in simon's dev

**Comment by brendanheywood** [comment]
What ended up happening here, was there a new type instead of 'free'? eg 'adjectival'?

**Comment by scd** [comment]
I have not added the new adjectival type. Because the rating system issues were resolved with the new rating system flag I did not make this change. It is likely we will make this change in the long run. I would like to spend a bit of time working out if there is any deeper meaning of 'Free' which will cause problems.

There are several grading systems which fit this category. 

Sorry I should have created another issue so we did not forget about it. Now done - #2541
